packages/conf-bmake/conf-bmake.1.0/files/detect_program.sh

```diff
@@ -0,0 +1,28 @@
+### detect_program.sh -- Detect program and save its path
+
+detect_which()
+{
+    ( IFS=':'
+      for path_elt in ${PATH}; do
+          if [ -x "${path_elt}/$1" ]; then
+              printf '%s' "${path_elt}/$1"
+              exit 0
+          fi
+      done
+
+      printf 'false'
+      exit 1 )
+}
+
+detect_config()
+{
+    local program_path
+
+    program_path=$(detect_which "$1")
+    cat <<EOF
+path: "${program_path}"
+EOF
+}
+
+mkdir -p "$1"
+detect_config "$2" > "$1/opam.config"
```

Comment:
I believe @samoht is suggesting that you could add a static config file to this package itself. I'm not sure how this works in conjunction with dynamically detecting the appropriate `bmake` (or `make` if on BSD) path to invoke.